service/src/main/java/org/mskcc/cmo/metadb/service/SampleService.java

```diff
@@ -23,4 +23,6 @@
     List<SampleMetadata> getSampleMetadataListByCmoPatientId(String cmoPatientId) throws Exception;
 
     List<MetaDbSample> getAllMetadbSamplesByRequestId(String requestId) throws Exception;
+    
+    List<SampleMetadata> getSampleMetadataHistory(String igoSampleId) throws Exception;
```

Comment:
`getSampleMetadataHistory` --> `getSampleMetadataHistoryByIgoId`

It's good to name methods such that they describe what they're doing and *how* (when applicable). In this case we'd want to make it clear that we're searching by an igo id since samples may have more than one alias that we can search with